frontend/components/software/edit/editSoftwareConfig.ts

```diff
@@ -210,7 +226,7 @@ export const organisationInformation = {
     title: 'Add organisation',
     subtitle: 'We search organisation name in RSD and ROR databases',
```

Comment:
While I am at it, here is one more suggestion :)

```suggestion
    subtitle: 'We search for organisation names in the RSD and the ROR databases',
```